src/setup.erl

```diff
@@ -101,7 +141,7 @@ enable_cluster_int(Options, no) ->
         Port ->
             config:set("httpd", "port", integer_to_list(Port))
     end,
-    io:format("~nEnable Cluster: ~p~n", [Options]).
+    couch_log:notice("~nEnable Cluster: ~p~n", [Options]).
```

Comment:
There is no need in leading/trailing `~n` when use couch_log as it already sets all new lines breaks right.